config/environments/production.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require 'active_support/core_ext/integer/time'
+
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
+
+  # Code is not reloaded between requests.
+  config.cache_classes = true
+
+  # Eager load code on boot. This eager loads most of Rails and
+  # your application in memory, allowing both threaded web servers
+  # and those relying on copy on write to perform better.
+  # Rake tasks automatically ignore this option for performance.
+  config.eager_load = true
+
+  # Full error reports are disabled and caching is turned on.
+  config.consider_all_requests_local       = false
+  config.action_controller.perform_caching = true
```

Comment:
##[OPTIONAL]
* I strongly suggest you fix this `cors` error I get while trying to display random greetings on the frontend 
![image](https://user-images.githubusercontent.com/52497246/198025541-476cc9f4-a97b-4235-b199-9580fe19f0c2.png)
* I mark this as optional because I see that this bug was missed by previous CRs but it would be nice to fix this